config/router-chatbot.js

```diff
@@ -34,10 +37,19 @@ router.post('/', (req, res) => {
     return app.locals.controllers.donorsChooseBot.chatbot(req, res);
   }
 
-  const mobilecommonsOip = process.env.CAMPAIGNBOT_MOBILECOMMONS_OIP;
-  if (!mobilecommonsOip) {
-    logger.error('CAMPAIGNBOT_MOBILECOMMONS_OIP undefined');
+  let configured = true;
+  // Check for required config variables.
+  const settings = ['MOBILECOMMONS_OIP_CAMPAIGNBOT', 'MOBILECOMMONS_OIP_AGENTVIEW'];
+  settings.push('MOBILECOMMONS_KEYWORD_CAMPAIGNBOT');
+  settings.concat(['GAMBIT_CMD_MEMBER_SUPPORT', 'GAMBIT_CMD_REPORTBACK']);
+  settings.forEach((configVar) => {
+    if (!process.env[configVar]) {
+      logger.error(`undefined process.env.${configVar}`);
+      configured = false;
```

Comment:
I did write it that way at first -- but thought it could be useful to log each missing variable too.